### DISCUSSION TITLE
Plugin for a YT series

### DISCUSSION THREAD

**Discussion by Lunitar8** [runelite/runelite#16877] [Ideas]
Willing to pay 20€ for a plugin!

Hi! I would need a plugin specific for my youtube series, CaChunk. The plugin would be an infobox that counts combat achievement points, and the player can use those points to unlock new chunks via the Chunklocked-plugin. My latest video (CaChunk #3) on YT has a videoedited counter on the bottom of the screen which is an example of what i mean/need.

**Comment by digiholic** [comment]
This is actually quite similar to a plugin I'm already working on that unlocks chunks based on quest points, skill totals, and tasks.

If no one has jumped on this by Saturday I should be able to make something at least functionally useable over the weekend

**Reply by Lunitar8** [reply]
Sounds great! You can DM me on discord Lunitar#6138 if you want to discuss it more, and I can send a payment if you want it :)
